packages/ndla-ui/src/Resource/BlockResource.tsx

```diff
@@ -166,23 +166,33 @@ const BlockResource = ({
 }: Props) => {
   const firstResourceType = resourceTypes?.[0]?.id ?? "";
 
+  const contentType = useMemo(() => {
+    if (!firstResourceType) {
+      return MISSING;
+    } else {
+      return (
+        contentTypeMapping[firstResourceType] ??
+        resourceEmbedTypeMapping[firstResourceType] ??
+        contentTypeMapping["default"]
+      );
+    }
+  }, [firstResourceType]);
```

Comment:
Blir dette riktig? Gir det mer mening å falle tilbake til `missing` hvis man bommer på de to første?

Reply:
Tja, det er gjort slik for å unngå at det går i beina på eksisterende default-logikk. firstResourceType vil være en tom streng (falsy) når den ikke finnes, og det vil alltid (?) bety at den ikke er tilgjengelig (og vil aldri matche de første 2 i aller tilfelle). Derfor kan jeg plukke den ut først, også fordi jeg syns det gjør funksjonen lettere å lese.

Deretter, hvis den finnes, sjekke om den er i en av de 3 andre (som har ligget der tidligere), inkludert default (som jeg har tenkt at p.t. returnerer SUBJECT_MATERIAL fordi det er content types som ikke har sitt eget, enten nå eller senere om vi legger inn nye ting, folder er f.eks. på vei). Om jeg ikke ser helt feil, vil det gjelde SUBJECT, TOPIC, MULTIDISIPLINARY_TOPIC og EXTERNAL, p.t. gitt at noen av de typene kan favorittmerkes? Også som sagt folder, da, om ikke den får egne farger før den legges til. Har iallefall tenkt at det er mest robust om default ligger der den gjør.